src/views/create_story.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <v-container>
+    <div v-if="step === 1">
+      <h3>Select Genre</h3>
+      <v-row>
+        <v-col v-for="genre in genres" :key="genre.genre_id" cols="12" md="4">
+          <div
+            :class="{'selected-card': selectedGenre === genre, 'card': true}"
+            @click="selectGenre(genre)"
+          >
```

Comment:
why is this greater than sign on its own line?  stylistically, we typically keep it at the end of the previous line